Formula/neovim.rb

```diff
@@ -0,0 +1,90 @@
+class Neovim < Formula
+  desc "Ambitious Vim-fork focused on extensibility and agility"
+  homepage "https://neovim.io"
+
+  stable do
+    url "https://github.com/neovim/neovim/archive/v0.2.0.tar.gz"
+    sha256 "72e263f9d23fe60403d53a52d4c95026b0be428c1b9c02b80ab55166ea3f62b5"
+
+    resource "luarocks" do
+      url "https://github.com/luarocks/luarocks.git",
+        :revision => "5d8a16526573b36d5b22aa74866120c998466697"
```

Comment:
The "rule" we standardized on is alignment directly under " so for example
```
  url "https://github.com/facebook/osquery.git",
      :tag => "2.4.6",
      :revision => "f9cb7149a9ac53c4aa563886d3bd37955876753f"
```

```
  go_resource "github.com/mitchellh/gox" do
    url "https://github.com/mitchellh/gox.git",
        :revision => "c9740af9c6574448fd48eb30a71f964014c7a837"
  end
```
and for commands alignment under the `"` of the first or second argument depending on the phase of the moon. For example,
```
      system "go-bindata", "-pkg", "docker", "-nocompress", "-nomemcopy",
                           "-nometadata", "-o",
                           "#{dir}/executors/docker/bindata.go",
                           "prebuilt-x86_64.tar.xz",
                           "prebuilt-arm.tar.xz"
```
and
```
    system "go", "build", "-ldflags",
           "-X github.com/mholt/caddy/caddy/caddymain.gitTag=#{version}",
           "-o", bin/"caddy", "github.com/mholt/caddy/caddy"
```